package.json

```
@@ -65,9 +65,9 @@
     "build:int": "env-cmd -f .env.int yarn build",
     "build:development": "env-cmd -f .env.development yarn build",
     "build:demo": "env-cmd -f .env.demo yarn build",
-    "test:ci": "cross-env CI=true react-scripts test --env=jest-environment-jsdom-sixteen",
-    "test:related": "react-scripts test --env=jest-environment-jsdom-sixteen --watchAll=false -b --findRelatedTests",
-    "test": "react-scripts test --env=jest-environment-jsdom-sixteen",
+    "test:ci": "cross-env CI=true react-scripts test --env=jest-environment-jsdom-sixteen --watchAll=true --testTimeout=120000",
```

In CI testing I don't think it's appropiate to have a watch all true, here we want the process to start and finish.
```suggestion
    "test:ci": "cross-env CI=true react-scripts test --env=jest-environment-jsdom-sixteen --testTimeout=120000",
```